ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -334,19 +369,19 @@ Item {
                             return
                         }
 
-                        if (root.rootStore.sendMessage(d.activeChatContentModule.getMyChatId(),
-                                                       event,
-                                                       chatInput.getTextWithPublicKeys(),
-                                                       chatInput.isReply? chatInput.replyMessageId : "",
-                                                       chatInput.fileUrlsAndSources
-                                                       ))
-                        {
-                            Global.playSendMessageSound()
-
-                            chatInput.textInput.clear();
-                            chatInput.textInput.textFormat = TextEdit.PlainText;
-                            chatInput.textInput.textFormat = TextEdit.RichText;
-                        }
+                         if (root.rootStore.sendMessage(d.activeChatContentModule.getMyChatId(),
```

Comment:
can you apply this to the whole method? :)

Reply:
lol, I don't know what actually happened here!
But please forgive me, cause I'm gonna merge it as is. Don't want to re-run all CI jobs just because of this. Let's save some electricity 😄

Reply:
I think it was `CMD+I` :) as you wish